src/OrchardCore/OrchardCore.Users.Abstractions/UserOptions.cs

```diff
@@ -0,0 +1,27 @@
+namespace OrchardCore.Users
+{
+    public class UserOptions
+    {
+        private string _loginPath = "Login";
+        private string _logoffPath = "/Users/LogOff";
+        private string _changePasswordUrl = "ChangePassword";
```

Comment:
Not sure if this needs leading slash (I know it currently has one)

Can you check if it is good without, and if so remove it, and the trim can then be the same across them all?

Reply:
It works without the leading /.